chapters/testing.Rmd

```diff
@@ -1095,7 +1036,7 @@ TDD's advocates claim that this leads to better code because:
 
 These arguments are plausible.
 However,
-studies such as @Fucc2016 and @Fucc2017 don't support them:
+studies such as [@Fucc2016; @Fucc2017] don't support them:
```

Comment:
I think the original formatting was correct:
"studies such as Fucci et al. (2016) and Fucci et al. (2017) don’t..."

You've changed it to:
"studies such as (Fucci et al. 2016, 2017) don’t..."

Reply:
:+1: